Sources/ArcGISToolkit/Components/Augmented Reality/CalibrationView.swift

```diff
@@ -0,0 +1,180 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ARKit
+import SwiftUI
+import ArcGIS
+
+extension WorldScaleGeoTrackingSceneView {
+    /// A view that allows the user to calibrate the heading of the scene view camera controller.
+    struct CalibrationView: View {
+        /// The camera controller heading.
+        @Binding var heading: Double
+        /// The camera controller elevation.
+        @Binding var elevation: Double
+        /// A Boolean value that indicates if the user is calibrating.
+        @Binding var isCalibrating: Bool
+        /// The initial camera controller elevation.
+        @Binding var initialElevation: Double
+        /// The elevation delta value after calibrating.
+        @State private var elevationDelta = 0.0
```

Comment:
Are all of these values needed? It seems that the calibration view only needs to give a delta, I don't think it would actually need to know about the actual heading or elevation. Only the delta values.

Reply:
These values are necessary for the stepper control logic since the elevation delta value does not change.